Examples/Io/Json/include/ActsExamples/Io/Json/JsonMaterialWriter.hpp

```diff
@@ -39,11 +39,21 @@ namespace ActsExamples {
 /// using the Json Geometry converter
 class JsonMaterialWriter {
  public:
+  struct Config {
+    /// The config class of the converter
+    Acts::MaterialMapJsonConverter::Config converterCfg;
+    /// Output file name
+    std::string fileName = "material";
+    /// Write the material map as a Json file
+    bool writeJson = true;
+    /// Write the material map as a CBOR file
+    bool writeCbor = false;
```

Comment:
Just a suggestion, but how about making this an enum? That way, we could add support other JSON-like formats if we want to later on. We do have a similar enum in the Examples actually.